onnxruntime/core/optimizer/propagate_cast_ops.cc

```diff
@@ -79,14 +79,20 @@ static std::unordered_map<std::string, std::vector<int>> opcode_to_input_map = {
     {"Gather", {0}},
     {"Reshape", {0}},
     {"Dropout", {0}},
+    {"Expand", {0}},
     {"LayerNormalization", {0, 1, 2}},
+    {"Squeeze", {0}},
+    {"Unsqueeze", {0}}
```

Comment:
so if an op is missing in this list, all the inputs and outputs would be considered? Is that why some ops like Add are missing here?

Reply:
If an op is missing then it means that all inputs/outputs need to be considered. Whereas for certain ops different inputs/outputs are not floating point data and/or not relevant.